lib/maps.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+var Stream = require('stream')
+  , util = require('util')
+  , request = require('request')
+  , debug = require('debug')('cartodb')
+  , resources = require('./resources.json')
+  , Mustache = require('mustache')
+  , _Promise = require('./promise')
+  ;
+
+/*
+ * Inheritance
+ */
+
+util.inherits(Named, Stream);
+
+
+function Named(config) {
+
+  for(var i in config){
+    this[i] = config[i];
+  }
+
+  this.maps_api_url = this.maps_api_url || Mustache.render(resources.maps_api_url, config);
```

Comment:
This code is the same as in SQL and Import constructors. Would be nice to inherit from some base class.